components/Form/index.js

```diff
@@ -0,0 +1,332 @@
+import styled from "styled-components";
+import { css } from "styled-components";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+export default function Form({ onSubmit }) {
+  const [sleep, setSleep] = useState(true);
+  const [hygiene, setHygiene] = useState(false);
+  const [medicine, setMedicine] = useState(false);
+
+  const router = useRouter();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    for (const property in data) {
+      if (data[property].trim() === "") {
+        return alert("Bitte gib einen Text in das Feld ein!");
+      }
```

Comment:
Here you are checking if the user entered only whitespace in the input fields. But you do not trim the input before adding it to your `localStorage`. So you can end up with entries like "    test-facility", which is not really, what you want, I guess 😄 

Additionally, it would be good to give the user a hint in the alert, where he/she needs to make changes in order to submit the form successfully. Right now, the same message is displayed for all input fields. You might be able to use `property` in your alert-message to display more detailed instructions for the user.